### PR TITLE
Enhancement of the Application

### PR DESCRIPTION
**Pull request by Yashbanu**
- **Big Increment** were changed to **Increment** in schema definition
- **Comments** were added to the table migration for better understanding
- **VARCHAR length** were defined in migration
- **Empty lines** in controller were removed
- **Function** in controller were **renamed** such that it starts with small letter 
- **Timestamp()** were changed to true

### CODE REVIEW

File: form/database/migrations/2024_01_16_122809_create_products_table.php [hunk 1]
```diff
@@ -13,11 +13,11 @@
     public function up(): void  
     {
         Schema::create('products', function (Blueprint $table) {
-            $table->id();
-            $table->string('name');
-            $table->string('category');
-            $table->integer('price');
-            $table->integer('quantity');
+            $table->increments('id')->comment("Unique ID for each product");
+            $table->string('name',50)->comment("Name of the product");
```
Comment (Naveen1895) [inline form/database/migrations/2024_01_16_122809_create_products_table.php:17]: @Yashbanu is 50 and 10 enough for name and category ?

File: form/database/seeders/ProductCollection.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Console\Seeds\WithoutModelEvents;
 use Illuminate\Database\Seeder;
 use Illuminate\Support\Facades\DB;
-
+use App\Models\Product;
```
Comment (Naveen1895) [inline form/database/seeders/ProductCollection.php:8]: @Yashbanu why do we need this import ?

Comment (Yashbanu) [reply form/database/seeders/ProductCollection.php:8]: @Naveen1895  Initially I thought of using Eloquent to insert data so i used import later I used DB facade to insert data directly . It is not necessary now , I will remove it.

File: form/database/migrations/2024_01_16_122809_create_products_table.php (hunk 1 above)
Comment (Yashbanu) [reply form/database/migrations/2024_01_16_122809_create_products_table.php:17]: @Naveen1895 I have given mine own set of category in blade file using radio button whose length is less than 10 so I restricted the length as 10. If necessary I can make a change on it.

File: form/database/seeders/ProductCollection.php (hunk 2 above)
Comment (Naveen1895) [reply form/database/seeders/ProductCollection.php:8]: Okay

**Review by Naveen1895 — APPROVED** [review]

**Pull request merged**